lib/apiservers/engine/backends/backends.go

```diff
@@ -310,3 +313,23 @@ func setPortMapping(info *models.ContainerInfo, backend *Container, container *c
 	}
 	return nil
 }
+
+func loadRegistryCACerts() {
+	if len(vchConfig.RegistryCertificateAuthorities) == 0 {
+		return
+	}
+
+	rootCertPool, err := x509.SystemCertPool()
+	log.Debugf("Loaded %d CAs for registries from system CA bundle", len(rootCertPool.Subjects()))
+	if err != nil {
+		log.Fatalf("Unable to load system CAs")
+	}
+
+	if !rootCertPool.AppendCertsFromPEM(vchConfig.RegistryCertificateAuthorities) {
+		log.Fatalf("Unable to load CAs for registry access in config")
```

Comment:
Log an error - potentially write it to a file of major errors that vic-admin can display, but we shouldn't become completely non-functional because of this.